src/components/dashboard/SendByQR.web.js

```diff
@@ -46,7 +48,13 @@ const SendByQR = ({ screenProps }: Props) => {
   }
 
   const handleError = e => {
-    log.error('QR scan send failed', e.message, e)
+    showErrorDialog({
```

Comment:
we should check error object passed to the callback.
not all errors are camera errors. pls console log which error object (message, name code other properties) is passed exactly in the case if we've restricted camera access 

if i remember, it should be error.name === 'NotAllowed' or error.name === 'NotAllowedError'
Please, re-check this

Reply:
for other errors show dialog with text
"QR code scan failed" and also pass the e.message as second param
@hluzdanNord  btw this is not the correct syntax for showErrorDialog, please always look in other places in code how are things being done.

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1833/commits/5898f60773c28d05802d6f29d15a1b171a6054df